### PR TITLE
Update startup-program.md

### PR DESCRIPTION
**Pull request by cerisemiller-mixpanel**
updates to Mixpanel for Startups program docs

### CODE REVIEW

**Review by junhouse — APPROVED** [review]
LGTM

File: pages/docs/pricing/startup-program.md [hunk 1]
```diff
@@ -1,106 +1,84 @@
...
 
-- A monthly growth plan at the 100,000 MTU rate would work out to a bill of $1190/month*
-- Each additional MTU above the subscription limit of 100,000 per month costs $0.01428
-- Since an additional 50,000 MTU (150,000 MTUs consumed on a 100,000 MTU plan) would be consumed each month, additional data charges will be $714/month
-- Over a 12-month period, the annual cost would sum to $22,848 and this would all be covered by your credit balance in the first year.*
+After acceptance to the program, $50,000 in credits were added to participants’ Mixpanel account and automatically applied to eligible line items when purchasing an annual [Growth plan](https://mixpanel.com/pricing/#edit-plan) via our plan-builder. The credits cover up to $50,000 in total usage within the first year. Any additional usage costs in excess of $50,000 is not covered by the program and will be charged to the credit card on file.
```
Comment (junhouse) [inline pages/docs/pricing/startup-program.md]: change `$50,000` -> `\$50,000`

Comment (junhouse) [reply pages/docs/pricing/startup-program.md]: I would do this for anywhere you have `$50,000`

Comment (cerisemiller-mixpanel) [reply pages/docs/pricing/startup-program.md]: thank you!

**Pull request merged**